### ISSUE TITLE
Half block shift for blocks on rotor

### ISSUE THREAD

**Issue by dhelonious** [tsunamayo/Starship-EVO#2344]
### Short Description of the issue:

The grid for small blocks is shifted by a half block on two sides of a small block, which is attached to the base of a rotor. While such a placement is not useful in the first place, I still think this is a bug.

### Steps to reproduce the issue:

Build a rotor block and attach a small block to the base of the rotor (see screenshots below).

### Environment
- Starship EVO version: pre-alpha 20w23c

### Additional information:

Screenshots:
![20200605234338_1](https://user-images.githubusercontent.com/6987220/83925130-b3125200-a786-11ea-9bfd-1802a97ae193.jpg)
![20200605234353_1](https://user-images.githubusercontent.com/6987220/83925134-b60d4280-a786-11ea-9cb1-9970ed22708f.jpg)
![20200605234356_1](https://user-images.githubusercontent.com/6987220/83925140-b86f9c80-a786-11ea-8c75-de88b018c153.jpg)

**Comment by tsunamayo** [comment]
Yes indeed, there is an issue as this brick is in fact build at a half brick height offset. I will have a look into this as this is not the expected behaviour.
